test/components/form/submission.spec.js

```diff
@@ -184,4 +187,36 @@ describe('FormSubmission', () => {
           }));
     });
   });
+
+  describe('session timeout', () => {
+    beforeEach(() => {
+      mockLogin();
+    });
+
+    it('does not auto log out', async () => {
+      testData.extendedForms.createPast(1, { xmlFormId: 'a', webformsEnabled: true });
+      const session = testData.sessions.createNew({ expiresAt: '1970-01-01T00:05:00Z' });
+
+      const clock = sinon.useFakeTimers();
```

Comment:
I usually call `sinon.useFakeTimers()` before creating `testData` so that the default timestamps that `testData` uses reflect the faked time.